grails-app/views/content/index.gsp

```diff
@@ -29,6 +29,8 @@
 		<link id="imgCFMP" rel="prefetch" href="${resource(dir: 'images/content', file: 'knowDimCFMP.png')}">
 
 		<g:javascript src="source/topicDialog.js" defer="defer" />
+		<g:external dir="js/plugins" file="jquery.piegraph.js"/>
+		<g:external dir="js/plugins" file="jquery.canvasjs.min.js"/>
```

Comment:
files in the plugins folder are not check into git.
Bower manages loading in these files.
When adding a new plugin add it to the `bower.json` file in the root of the project.